corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -656,10 +673,9 @@ def get_query_datums(self, module, uses_inline_search):
         """
         from ..post_process.remote_requests import (
             RESULTS_INSTANCE,
-            RESULTS_INSTANCE_INLINE,
             RemoteRequestFactory,
         )
-        storage_instance = RESULTS_INSTANCE_INLINE if uses_inline_search else RESULTS_INSTANCE
+        storage_instance = module.search_config.get_instance_name() if uses_inline_search else RESULTS_INSTANCE
```

Comment:
Would it be advisable to move this `if/else` logic into `get_instance_name()`, either by passing in `uses_inline_search` or looking it up in the body of that method.  Looks like this logic is applied in each of the references to it, though maybe with subtle differences that matter, I'm not certain

Reply:
I considered it, but the `else` value isn't always `RESULTS_INSTANCE`, and there's a unique logic for each of the callers to decide what value to assign.
For example in this  [usage](https://github.com/dimagi/commcare-hq/blob/bfbf12698f53958dbbc137c21dc9bb197b8d6143/corehq/apps/app_manager/suite_xml/sections/entries.py#L250), that `else` value would need to be passed into `get_instance_name()` or a flag to know to use `casedb` instead of `RESULTS_INSTANCE` which I don't think is readable.